targets/TARGET_ublox/TARGET_R5030/TARGET_R5XXX/PinNames.h

```diff
@@ -100,10 +107,11 @@ typedef enum {
 typedef enum {
     PullNone = 0,
     PullDown = 1,
-    PullUp = 0xff,  // TODO: current HI2110 chip does not have pull-up 
+    PullUp = 2,  // TODO: current HI2110 chip does not have pull-up
     PullDefault = PullDown // TODO: mbed requires PullUp as default but HI2110 doesn't have it 
```

Comment:
Can you please update comment as PullDown is default

Reply:
ok